frontend/src/pages/CourseSelector/CourseMenu/CourseMenu.jsx

```diff
@@ -221,17 +224,19 @@ const MenuItem = ({
         onClick={handleClick}
       >
         <div className="menuItemContainer">
-          <div>
-            {`${courseCode} `}
-            {!accurate && (
-              <TooltipWarningIcon
-                text="We couldn't parse the requirement for this course. Please manually check if you have the correct prerequisites to unlock it."
-              />
-            )}
-            {!unlocked && <LockOutlined style={{ fontSize: "11px" }} />}
-          </div>
+          <Tooltip title={`${courseTitle}`} placement="topLeft">
```

Comment:
Could do 'title={courseTitle}'?

Reply:
<img width="504" alt="image" src="https://user-images.githubusercontent.com/97064025/170426910-6fa91cb3-d8a7-453f-bccf-9f9978b7d1d3.png">
I get this error..